apps/wallet/src/ui/app/components/menu/content/Account.tsx

```diff
@@ -21,31 +22,42 @@ export function Account({ account }: AccountProps) {
     const copyCallback = useCopyToClipboard(address, {
         copySuccessMessage: 'Address copied',
     });
+    const { data: suins } = useResolveSuiNSName(address);
```

Comment:
I think we also need to add support for resolving SuiNS names in the import Ledger accounts flow (`LedgerAccountRow.tsx`), but we can do that and other potential misses in a follow-up

Reply:
Yeah, I probably missed a bunch of cases, I'll see if there's an easy way to add it to the import flow (we probably _also_ want to show the raw address there too)